src/app/state/reducers/eventsState.ts

```diff
@@ -1,7 +1,7 @@
 import { Action, AugmentedEvent, EventMapData, EventOverview, NOT_FOUND_TYPE } from "../../../IsaacAppTypes";
 import { ACTION_TYPE, NOT_FOUND } from "../../services";
-import { EventBookingDTO } from "../../../IsaacApiTypes";
 import { routerPageChange } from "../index";
+import { DetailedEventBookingDTO } from "../../../IsaacApiTypesGenerated";
```

Comment:
This shouldn't be imported directly from this file. Now this is making me wonder if there is a way to ensure we don't accidentally export directly from this file or from the custom one... 🤔

Reply:
Looks like that was the only incident. I'm not sure of a way to prevent it from happening though, other than being more careful not to make silly mistakes 😀